### PR TITLE
breaking: video compression false

### LINKED ISSUE #26160: Turn off videoCompression by default
### What would you like?

Cypress also automatically compresses the recorded video to a Constant Rate Factor (CRF) of 32. Cypress could turn off video compression by default for all users - having a `videoCompression`  value set to `false`  in their configuration file.

### Why is this needed?

Recording video in low CPU resourced machines results in slower runtime of Cypress tests. Video compression is on my default and blocking, so it increases the time the tests take to run. And video compression takes longer the longer the video is, so these reasons are compounding the problem. 

### Other

Turning off video compression would result in larger video sizes, which still needs exploration as to the impact for users and Cypress.

The release and implementation strategy has not been decided, but this could result in a breaking change for users.

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes #26160

### Additional details
For Cypress v13 , video will be set to false by default. In addition, we will also be setting `videoCompression` to false. This PR implements those changes.

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? https://github.com/cypress-io/cypress-documentation/pull/5301
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: system-tests/projects/react-vite-ts-configured/cypress.config.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/react-vite-ts-configured/cypress.config.ts]:
```suggestion
```

File: system-tests/projects/svelte-vite/cypress.config.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/svelte-vite/cypress.config.js]:
```suggestion
```

File: system-tests/projects/ts-proj-4-5/cypress.config.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/ts-proj-4-5/cypress.config.ts]:
```suggestion
```

File: system-tests/projects/webpack-dev-server-ts/cypress.config.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/webpack-dev-server-ts/cypress.config.ts]:
```suggestion
```

File: system-tests/projects/vue3-vite-ts-custom-index-html/cypress.config.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/vue3-vite-ts-custom-index-html/cypress.config.ts]:
```suggestion
```

File: system-tests/projects/vue3-vite-ts-configured/cypress.config.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/vue3-vite-ts-configured/cypress.config.ts]:
```suggestion
```

File: system-tests/projects/vite-simple/cypress.config.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/vite-simple/cypress.config.js]:
```suggestion
```

File: system-tests/projects/ts-proj-5/cypress.config.ts [hunk 8]
```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'cypress'
 
 export default defineConfig({
-  videoCompression: false, // turn off video compression for CI
+
```
Comment (emilyrohrbough) [inline system-tests/projects/ts-proj-5/cypress.config.ts]:
```suggestion
```

**Review by emilyrohrbough — APPROVED** [review]

**Comment by AtofStryker** [comment]
@emily the compression config removal, even though a no-op here, is handled in https://github.com/cypress-io/cypress/pull/27009

**Comment by AtofStryker** [comment]
weird thought I committed these 😅

**Review by chrisbreiding — APPROVED** [review]

File: cli/CHANGELOG.md [hunk 9]
```diff
@@ -7,6 +7,7 @@ _Released 8/1/2023 (PENDING)_
 
 - The [`cy.readFile()`](/api/commands/readfile) command is now retry-able as a [query command](https://on.cypress.io/retry-ability). This should not affect any tests using it; the functionality is unchanged. However, it can no longer be overwritten using [`Cypress.Commands.overwrite()`](/api/cypress-api/custom-commands#Overwrite-Existing-Commands). Addressed in [#25595](https://github.com/cypress-io/cypress/pull/25595).
 - the [video](https://docs.cypress.io/guides/references/configuration#Videos) configuration option will now default to `false`. Addresses [#26157](https://github.com/cypress-io/cypress/issues/26157).
+- The [`videoCompression`](https://docs.cypress.io/guides/references/configuration#Videos) configuration option will now default to `false`. Addresses [#26160](https://github.com/cypress-io/cypress/issues/26160).
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]:
```suggestion
- The [`videoCompression`](https://docs.cypress.io/guides/references/configuration#Videos) configuration option now defaults to `false`. Addresses [#26160](https://github.com/cypress-io/cypress/issues/26160).
```

File: cli/CHANGELOG.md [hunk 10]
```diff
@@ -7,6 +7,7 @@ _Released 8/1/2023 (PENDING)_
 
 - The [`cy.readFile()`](/api/commands/readfile) command is now retry-able as a [query command](https://on.cypress.io/retry-ability). This should not affect any tests using it; the functionality is unchanged. However, it can no longer be overwritten using [`Cypress.Commands.overwrite()`](/api/cypress-api/custom-commands#Overwrite-Existing-Commands). Addressed in [#25595](https://github.com/cypress-io/cypress/pull/25595).
 - the [video](https://docs.cypress.io/guides/references/configuration#Videos) configuration option will now default to `false`. Addresses [#26157](https://github.com/cypress-io/cypress/issues/26157).
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]:
```suggestion
- The [`video`](https://docs.cypress.io/guides/references/configuration#Videos) configuration option nows default to `false`. Addresses [#26157](https://github.com/cypress-io/cypress/issues/26157).
```

**Pull request merged**